PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx

```diff
@@ -2953,6 +2967,8 @@ void AliAnalysisTaskGammaConvCalo::UserCreateOutputObjects(){
         if (fIsMC > 1){
           fHistoTruePi0InvMassPt[iCut]->Sumw2();
           fHistoTrueEtaInvMassPt[iCut]->Sumw2();
+          fHistoTruePi0InvMassPtAdditional[iCut]->Sumw2();
+          fHistoTrueEtaInvMassPtAdditional[iCut]->Sumw2();
```

Comment:
this needs to be in the "if (fDoLightOutput!=2 ){" condition

Reply:
Yes you are correct, thanks for noticing.
I moved the lines accordingly.